apps/admin_api/lib/admin_api/v1/controllers/token_controller.ex

```diff
@@ -159,7 +191,55 @@ defmodule AdminAPI.V1.TokenController do
   end
 
   def update(conn, _) do
-    handle_error(conn, :invalid_parameter, "Invalid parameter provided. `id` is required.")
+    handle_error(conn, :missing_id)
+  end
+
+  def get_erc20_capabilities(conn, %{"blockchain_address" => blockchain_address}) do
+    with :ok <- BlockchainHelper.is_blockchain_address(blockchain_address),
+         {:ok, _} <- authorize(:get_erc20_capabilities, conn.assigns, %Token{}),
```

Comment:
Can `%Token{}` be `nil` instead? This operation doesn't seem to depend on `%Token{}`. Similar to how we do `authorize(:all, conn.assigns, nil)` in other places.